### PR TITLE
Updated Upgrade Guide for 3.10 release

### PR DESCRIPTION
**Pull request by JeffSchering**

### CODE REVIEW

File: source/administration/upgrade.rst [hunk 1]
```diff
@@ -122,19 +122,22 @@ Version Archive
...
   - ``https://releases.mattermost.com/3.8.2/mattermost-3.8.2-linux-amd64.tar.gz``
   - SHA-256 Checksum: ``b99c86a2667f636eaee26331aa61a71a51b2d3d412eaa83fdebf8b53cddc6aeb``
-Mattermost Enterprise Edition v3.7.5 - `View Changelog <https://docs.mattermost.com/administration/changelog.html#release-v3-7-5>`_ - `Download <https://releases.mattermost.com/3.7.5/mattermost-3.7.5-linux-amd64.tar.gz>`_
-  - ``https://releases.mattermost.com/3.7.5/mattermost-3.7.5-linux-amd64.tar.gz``
-  - SHA-256 Checksum: ``65e65da661edbc7b7b2b02411f13dbe498fd704d5ae1289789feca79fe00b58a``
 
 Mattermost Enterprise Edition - Unsupported Versions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Mattermost Enterprise Edition v3.7.5 - `View Changelog <https://docs.mattermost.com/administration/changelog.html#release-v3-7-5>`_ - `Download <https://releases.mattermost.com/3.7.5/mattermost-3.7.5-linux-amd64.tar.gz>`_
```
Comment (esethna) [inline source/administration/upgrade.rst]: What qualifies a build as unsupported?

File: source/administration/upgrade.rst [hunk 2]
```diff
@@ -6,9 +6,9 @@ In most cases you can upgrade Mattermost Server in a few minutes, but the upgrad
 Which Upgrade Instructions to Use
 ---------------------------------
 
-If you are upgrading from version 3.7.0 or greater
+If you are upgrading from version 3.8.0 or greater
```
Comment (esethna) [inline source/administration/upgrade.rst]: What is the reasoning for changing this?

File: source/administration/upgrade.rst [hunk 3]
```diff
@@ -6,9 +6,9 @@ In most cases you can upgrade Mattermost Server in a few minutes, but the upgrad
...
   This guide applies to you. Skip to :ref:`Before you begin <before-you-begin>`.
-If you are upgrading from version 3.0.3 or greater and 3.6.6 or lower
+If you are upgrading from version 3.0.3 or greater and 3.7.5 or lower
```
Comment (esethna) [inline source/administration/upgrade.rst]: What is the reasoning for changing this?

File: source/administration/upgrade.rst (hunk 2 above)
Comment (JeffSchering) [inline source/administration/upgrade.rst]: The way it's currently set up, supported versions are in this doc and unsupported versions are in the legacy doc.

File: source/administration/upgrade.rst (hunk 3 above)
Comment (JeffSchering) [inline source/administration/upgrade.rst]: The way it's currently set up, supported versions are in this doc and unsupported versions are in the legacy doc.

File: source/administration/upgrade.rst (hunk 1 above)
Comment (JeffSchering) [inline source/administration/upgrade.rst]: Supported builds are the latest dot release + 2 releases back. When 3.10 is released, only 3.9 and 3.8 are supported.

Comment (esethna) [inline source/administration/upgrade.rst]: Okay, thanks

**Pull request merged**